utils/fix_xc7_carry.py

```diff
@@ -0,0 +1,595 @@
+""" Script for addressing CARRY4 output congestion in elaborated netlists.
+
+Usage:
+
+    python3 fix_carry.py < input-netlist-json > output-netlist-json
+
+Description:
+
+    In the 7-series SLICEL (and SLICEM) sites, there can be output congestion
+    if both the CO and O of the CARRY4 are used.  This congestion can be
+    avoided if transparent/open latches or registers on the output of the
```

Comment:
This congestion can be avoided by using a transparent/open latch or register on the output of the CARRY4.

Reply:
Fixed